spec/requests/api/v1/student_mentors/student_mentor_get_all_request_spec.rb

```diff
@@ -0,0 +1,76 @@
+require 'rails_helper'
+require './lib/tokenator.rb'
+include Tokenator
+
+describe 'student_mentors API' do
+  describe 'GET /api/v1/student_mentors' do
+    describe 'as an admin user' do
+      it 'returns all student_mentors in db' do
+        user = User.create(name: "name_of_user", role: "admin", login: "xyz")
+        #token = Tokenator.encode(user.login)
+
+        students = create_list(:student, 4)
+        mentors = create_list(:mentor, 2)
+
+        mentor_1 = mentors[0]
+        mentor_2 = mentors[1]
+
+        student_1 = students[0]
+        student_2 = students[1]
+        student_3 = students[2]
+        student_4 = students[3]
+
+        student_mentor_1 = StudentMentor.create(student_id: student_1.id, mentor_id: mentor_1.id, active: true)
```

Comment:
`active` also defaults to true when created, is that correct? Maybe active doesn't need to be assigned in setup.